include/odp/api/spec/packet_io.h

```diff
@@ -159,6 +159,34 @@ typedef struct odp_pktin_queue_param_ovr_t {
 	odp_schedule_group_t group;
 } odp_pktin_queue_param_ovr_t;
 
+/**
+ * Packet input vector configuration
+ */
+typedef struct odp_pktin_vector_config_t {
```

Comment:
I think we need to have a enable flag here to indicate that the vector config is valid. Makes config validation simpler,

Reply:
Makes sense. I will add "enable" flag in next version.